### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by hiergaut**
added contributing.md file from template

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  - [Improving The Documentation](#improving-the-documentation)
+- [Styleguides](#styleguides)
+  - [Commit Messages](#commit-messages)
+- [Join The Project Team](#join-the-project-team)
+
+
+
+## I Have a Question
+
+> If you want to ask a question, we assume that you have read the available [Documentation](https://github.com/T-PLAY/hub/doc).
```
Comment (nmellado) [inline .github/CONTRIBUTING.md:31]: This page is not available yet.
We need to have a PR with doc deployment before.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+- Open an [Issue](https://github.com/T-PLAY/hub/issues/new). (Since we can't be sure at this point whether it is a bug or not, we ask you not to talk about a bug yet and not to label the issue.)
+- Explain the behavior you would expect and the actual behavior.
+- Please provide as much context as possible and describe the *reproduction steps* that someone else can follow to recreate the issue on their own. This usually includes your code. For good bug reports you should isolate the problem and create a reduced test case.
+- Provide the information you collected in the previous section.
+
+Once it's filed:
+
+- The project team will label the issue accordingly.
+- A team member will try to reproduce the issue with your provided steps. If there are no reproduction steps or no obvious way to reproduce the issue, the team will ask you for those steps and mark the issue as `needs-repro`. Bugs with the `needs-repro` tag will not be addressed until they are reproduced.
```
Comment (nmellado) [inline .github/CONTRIBUTING.md:97]: Be sure the labels are created

**Comment by hiergaut** [comment]
All good ?

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (hiergaut) [reply .github/CONTRIBUTING.md:97]: Label created

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (nmellado) [inline .github/CONTRIBUTING.md:31]:
```suggestion
> If you want to ask a question, we assume that you have read the available [Documentation](https://t-play.github.io/hub/).
```

Comment (hiergaut) [reply .github/CONTRIBUTING.md:31]: You fixed this below

Comment (hiergaut) [reply .github/CONTRIBUTING.md:31]: Url is correct

Comment (hiergaut) [reply .github/CONTRIBUTING.md:31]: Should be with sphinx https://t-play.github.io/hub/api/index.html

Comment (hiergaut) [reply .github/CONTRIBUTING.md:31]: or https://t-play.github.io/hub/api/ (sphinx)
https://t-play.github.io/hub/ (doxygen)

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  - Stack trace (Traceback)
+  - OS, Platform and Version (Windows, Linux, macOS, x86, ARM)
+  - Version of the interpreter, compiler, SDK, runtime environment, package manager, depending on what seems relevant.
+  - Possibly your input and the output
+  - Can you reliably reproduce the issue? And can you also reproduce it with older versions?
+
+<!-- omit in toc -->
+#### How Do I Submit a Good Bug Report?
+
+> You must never report security related issues, vulnerabilities or bugs including sensitive information to the issue tracker, or elsewhere in public. Instead sensitive bugs must be sent by email to <>.
```
[the remark this answers is not in the document]
Comment (nmellado) [reply .github/CONTRIBUTING.md:84]:
```suggestion
> You must never report security related issues, vulnerabilities or bugs including sensitive information to the issue tracker, or elsewhere in public. Instead sensitive bugs must be sent by email to tplay@irit.fr.
```

[the remark this answers is not in the document]
Comment (hiergaut) [reply .github/CONTRIBUTING.md:84]: How can we subscribe to tplay@irit.fr mail ? (to receive mail and respond)

**Comment by hiergaut** [comment]
Waiting for the #26 done because needed sphinx api url (beauty doc url)

**Comment by hiergaut** [comment]
Ready for review.

**Comment by nmellado** [comment]
I'll approve it once we merged #26 👍

**Pull request merged**